### ISSUE TITLE
[NEW VERSION] 2.4.12

### ISSUE THREAD

**Issue by siegfriedweber** [stackabletech/hbase-operator#189]
**Which new version of Apache HBase should we support?**

2.4.12

**Additional information**

[Release notes](https://archive.apache.org/dist/hbase/2.4.12/RELEASENOTES.md)

**Changes required**

no

**Implementation checklist**

Please don't change anything in this list.
Not all of these steps are necessary for all versions.

- [ ] Update the Docker image
- [ ] Update documentation to include supported version(s)
- [ ] Update operator to support the new version (if needed)
- [ ] Update integration tests to test use the new versions (in addition or replacing old versions
- [ ] Update examples to use new versions

**Comment by maltesander** [comment]
From what i see there is no phoenix release for HBase 2.5.0 yet?
https://phoenix.apache.org/download.html
```
Current release 4.16.1 can run on Apache HBase 1.3, 1.4, 1.5 and 1.6.
Current release 5.1.2 can run on Apache HBase 2.1, 2.2, 2.3 and 2.4
```

**Comment by lfrancke** [comment]
Good point. In that case let's skip it for this release. I'm going to remove it from the board and the epic.
We can check again for our next release.

https://issues.apache.org/jira/browse/PHOENIX-6692 it'll be in Phoenix 5.2 which will be released soonish but not soon https://lists.apache.org/thread/85f16z83sfw056xolwvztb7yr9b0t9pg
